### ISSUE TITLE
RDD lineage support

### ISSUE THREAD

**Issue by karanglass34** [AbsaOSS/spline-spark-agent#33]
I am using the latest spline version. When I enable spline, I see incomplete lineage, my code reads the file from parquet and creates a temporary view, performs some transformations and creates a new view which is then finally written to a parquet. However I am not able to see lineages for my transformations. 

![lineage](https://user-images.githubusercontent.com/52342684/60380541-46ca9280-99fc-11e9-8167-577ffe84e9a4.png)

Are there any constraints like all the intermediate RDD's should be in memory? or any specific spark config apart from that mentioned in the documentation which needs to enabled?

**Comment by ashyamala** [comment]
@wajda Do you still feel that Spline is still on track to support user-defined RDD lineage or any kind of RDD lineage within the 0.6 release or before the end of the year?

**Comment by wajda** [comment]
@ashyamala, it's still on the road map, the demand for this feature is growing. But apparently it won't fit into 0.6, as we've got plenty of stuff to do in 0.6 already.

**Comment by wajda** [comment]
End of the year - I'd say very likely.
Or, of course if anybody from the community could help us with that it could be released earlier.

**Comment by wajda** [comment]
It didn't make it to 0.6. It doesn't seem to be a priority so far...

**Comment by elymberopoulos** [comment]
Hello, I'm just wondering what the state of this issue is? Is it actively in progress?

**Comment by wajda** [comment]
No, it's not a priority at the moment, but is still on the road map.
